added description string to the themeloader, this is then used to dis…

carlg0ransson opened the pull request:
…play the description of the theme after applying a new theme in the preferences.



----

- [ ] Change in CHANGELOG.md described
- [x] Manually tested changed features in running JabRef
- [x] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)

koppor commented on src/main/java/org/jabref/gui/preferences/AppearanceTabViewModel.java:
```diff
@@ -81,10 +81,10 @@ public void storeSettings() {
         }
 
         if (themeLightProperty.getValue() && !preferences.get(JabRefPreferences.FX_THEME).equals(ThemeLoader.MAIN_CSS)) {
-            restartWarnings.add(Localization.lang("Theme changed:") + " " + ThemeLoader.MAIN_CSS);
```
I would include the theme name directly in the localization: `Localization.lang("Theme changed to light theme");`. This way, the theme name is also translated. I would accept the two new localization strings.

koppor commented:
@carlg0ransson Could you check the items in the checklist at your PR? CHANGELOG.md entry. You can fill the checkbox with an x --> `- [x] checked`. You can remove items not necessary (e.g., Tests created for changes).

koppor commented:
@carlg0ransson Nice to see a contribution to JabRef's code base from someone new to JabRef's code.

carlg0ransson commented:
Hey, this is the first time I am contributing to an open source project - so I am sorry if I am not doing the whole PR process correctly. I've implemented the change you suggested now, and I've updated the initial comment with the checklist. I was not sure how/if it was needed to update the changelog for this small fix.

koppor commented:
The changelog could read: `Added speaking output at theme change. [#5584](https://github.com/JabRef/jabref/pull/5584)`

@tobiasdiez LGTM. WDYT?
